Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1569,6 +1575,8 @@ def gen2018HiMix(fragment,howMuch):
 steps['RECOHI2015']=merge([hiDefaults2015,{'-s':'RAW2DIGI,L1Reco,RECO,VALIDATION,DQM'},step3Up2015Defaults])
 steps['RECOHI2011']=merge([hiDefaults2011,{'-s':'RAW2DIGI,L1Reco,RECO,VALIDATION,DQM'},step3Defaults])
 
+steps['RECOPPREF2017']=merge([ppRefDefaults2017,step3Up2015Defaults])
```

Comment:
I notice that in the 94X backport PR (#21069) this line reads

steps['RECOPPREF2017']=merge([ppRefDefaults2017,{'-s':'RAW2DIGI,L1Reco,RECO,EI,PAT,VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@miniAODDQM'},step3Up2015Defaults])

i.e. the RECOSIM got removed: I think the same should be also done here

Reply:
@perrotta 
I don't understand your comment.
Please elaborate.

Reply:
While reviewing #21069 I noticed that this line differs from
https://github.com/cms-sw/cmssw/pull/21069/files#diff-819af2ec677272fddf1fef50e788a88bR1578

Reply:
I consider this PR the correct implementation https://github.com/cms-sw/cmssw/pull/21122#discussion_r149028129

Reply:
Ok, thank you Slava. I supposed it was done on purpose, instead.
Then it has to be fixed in the backport: I will add a note o that PR